src/screens/payment-methods/PaymentMethods.tsx

```diff
@@ -150,15 +155,84 @@ export default class PaymentMethods extends BaseAutoRefreshScreen<Props, State>
               count={count}
               limit={limit}
               skip={skip}
-              renderItem={(item: BillingPaymentMethod) => <PaymentMethodComponent paymentMethod={item} navigation={navigation} />}
+              renderItem={(paymentMethod: BillingPaymentMethod) => (
+                <Swipeable
+                  overshootRight={false}
+                  overshootLeft={false}
+                  childrenContainerStyle={style.swiperChildrenContainer}
+                  renderRightActions={() => this.renderSwipeableActions(paymentMethod, style)}>
```

Comment:
Rename method 'renderSwipeableActions' to a more meaningful action like 'renderDeletePaymentMethod'? ;-)